### ISSUE TITLE
Error integrating ExportSelectToolbarItem() in Toolbar

### ISSUE THREAD

**Issue by lagaguate** [elapouya/django-listing#19]
Hi,
First thanks for building django-list, good job.

I'm trying to replicate your example using ToolbarListing

I have discovered that when I include the ExportSelectToolbarItem(), when executed it gives me the error:

> Invalid block tag on line 29: 'trans', expected 'elif', 'else' or 'endif'. Did you forget to register or load this tag?
> 
> {% if item.has_submit_button %}
> 29 <button type="submit" class="btn btn-secondary btn-sm ml-1" data-empty-select-msg="{% trans "Please select a file format !" %}">{{ item. button_label }}</button>
> 30 {% endif %}

If I remove Export Select ToolbarItem(), I show the object data.

Do you have any idea if it's a bug or something I'm doing wrong.

In setting.py

```
LANGUAGE_CODE = 'en'

LOCALE_PATHS = (os.path.join(BASE_DIR, "locale"),)

#TIME_ZONE = "America/Guatemala" # Correction for the Timestamps

USE_I18N = True
```

I have listing.html template
```
{% load i18n %}
{% load django_listing %}
{% load static `%}```
.
.
.
.
{% render_listing ToolbarListing PteLovh %}
```

view.py 

```
class ToolbarListingView(ListingView):
    """ ToolbarListingView """
    template_name = "lovheader/listing.html"
    context_classes = (
        ToolbarListing,
        PteLovh,
    )  
```

listing.py

```
class ToolbarSimpleListing(SimpleListing):
    per_page = 8   
        
class ToolbarListing(ListingVariations):
    variations_classes = (
        ToolbarSimpleListing,
    )
    toolbar = Toolbar(SortSelectToolbarItem(), ExportSelectToolbarItem())
  
    per_page = 8
    paginator_has_first_last = True
    exclude_columns = "id"
```

**Comment by elapouya** [comment]
Hi, What version are you using ?

**Comment by lagaguate** [comment]
django-listing 0.7.9

Also right now I have Django 5, but he downgrades it to 4.2 and gives me the same error.

Since I was getting the same error, I returned at 5

Django                    5.0.2
django-autocomplete-light 3.11.0
django-bootstrap-icons    0.8.7
django-bootstrap4         24.1
django-crispy-forms       2.1
django-environ            0.11.2
django-extensions         3.2.3
django-filter             23.5
django-htmx               1.17.2
**django-listing            0.7.9**
django-widget-tweaks      1.5.0

**Comment by elapouya** [comment]
I just looked at my code, this is a bug, I forgot to load i18n in one of my templates.
I will fix that tomorrow

**Comment by lagaguate** [comment]
Thanks you

**Comment by elapouya** [comment]
Fixed in v0.7.10

**Comment by lagaguate** [comment]
That's good, I'll try again
Thanks you
